test/extensions/secondorder/diag_ggn/test_diag_ggn.py

```diff
@@ -89,3 +90,47 @@ def test_diag_ggn_mc(problem):
         backpack_res, backpack_res_mc_avg, atol=MC_ATOL, rtol=MC_RTOL
     )
     problem.tear_down()
+
+
+@pytest.mark.parametrize("problem", PROBLEMS, ids=IDS)
+def test_diag_ggn_mc_batch_light(problem):
+    """Test the MC approximation of individual diagonal of
+    Generalized Gauss-Newton/Fisher with few mc_samples (light version)
+
+    Args:
+        problem (ExtensionsTestProblem): Problem for extension test.
+    """
+    problem.set_up()
+
+    backpack_res = BackpackExtensions(problem).diag_ggn_exact_batch()
+    mc_samples = 3000
+    backpack_res_mc_avg = BackpackExtensions(problem).diag_ggn_mc_batch(mc_samples)
+    # sum the batch, and compare with exact mc
```

Comment:
Is this really done?

Reply:
Ah, do you mean the `# sum the batch, and compare with exact mc`? I think I've missed it. Will look into it now